cluster/common/flux/deploy_flux.sh

```diff
@@ -58,6 +58,9 @@ fi
 # back to the roor dir
 cd ../../../../
 
+echo "KUBECTL CONFIG GET-CONTEXTS"
```

Comment:
@samiyaakhtar I understand why we added this for debugging purposes in the deploy_kubediff.sh file but we probably don't need this here.